lib/image_optim/worker/class_methods.rb

```diff
@@ -32,6 +32,14 @@ def option_definitions
         @option_definitions ||= []
       end
 
+      def timeout_option
+        option(
+          :timeout,
+          nil,
```

Comment:
Insert `Integer` as next parameter so you don't need to add `NilClass` in option_parser.rb